src/axom/quest/SignedDistance.hpp

```diff
@@ -70,14 +70,16 @@ class SignedDistance
    * \param [in] surfaceMesh user-supplied surface mesh.
    * \param [in] isWatertight indicates if the surface mesh is closed.
    * \param [in] maxObjects max number of objects for spatial decomposition.
-   * \param [in] maxLevels max levels for spatial decomposition (optional).
+   * \param [in] maxLevels max levels for spatial decomposition.
+   * \param [in] computeSign indicates if distance queries should compute signs.
    * \note Default maxLevels is 5 if not specified.
```

Comment:
maxlevels is required since it is not defaulted in the header declaration here. So this note is confusing.

Reply:
Thanks @rhornung67 -- As you mentioned, there is no default, so this option is required.
I fixed this since the documentation was previously incorrect. 

(It might be confusing from the github view, but the change that you commented on was actually a removal of the word "optional").

Reply:
Thanks for fixing the comment. I think in some initial implementation `maxLevels` was indeed optional.  That was changed at some point but the documentation was not updated to reflect this change...